gestion_base_adresse/test/test_execute_functions.py

```diff
@@ -873,14 +873,14 @@ def test_update_point_adresse(self):
 
         # Valider le point 4
         sql = (
-            "UPDATE adresse.point_adresse SET a_valider=False "
+            "UPDATE adresse.point_adresse SET valide=False "
```

Comment:
```suggestion
            "UPDATE adresse.point_adresse SET valide=True "
```
Il faut inverser la logique